config/metrics.js

```diff
@@ -189,4 +189,12 @@ module.exports = {
 		dontPrint: true,
 		extract: (item) => item.isPrivate,
 	},
+	RequireLastPushApproval: {
+		extract: (item) => !!getBPRules(item)?.requireLastPushApproval,		
+		permissions: ['ADMIN'],
+	},
```

Comment:
i think maybe you'd want to rebase this branch on top of main, and remove the commits that are part of the other 2 PRs?